content/doc/developer/tutorial/extend.adoc

```diff
@@ -24,7 +24,7 @@ First, create a class called +HelloWorldAction+ in the same package as +HelloWor
 
 [source,java]
 ----
-package org.jenkinsci.plugins.sample;
+package io.jenkins.plugins.sample;
```

Comment:
This file has a handful of `org/jenkinsci` as well in file paths.

Reply:
Oops, thought I searched for all such but apparently I missed those. Will fix.